packages/@react-spectrum/label/chromatic/HelpText.chromatic.tsx

```diff
@@ -9,68 +9,100 @@
-import {storiesOf} from '@storybook/react';
 import {TextField} from '@react-spectrum/textfield';
 
-storiesOf('HelpText', module)
-  .addParameters({providerSwitcher: {status: 'positive'}})
-  .add(
-    'description',
-    () => render({description: 'Password must be at least 8 characters.'})
-  )
-  .add(
-    'error message',
-    () => render({errorMessage: 'Create a password with at least 8 characters.', validationState: 'invalid'})
-  )
-  .add(
-    'disabled',
-    () => render({description: 'Password must be at least 8 characters.', isDisabled: true})
-  )
-  .add(
-    'labelAlign: end',
-    () => render({
-      description: 'Password must be at least 8 characters.',
-      labelAlign: 'end'
-    })
-  )
-  .add(
-    'labelPosition: side',
-    () => render({
-      description: 'Password must be at least 8 characters.',
-      labelPosition: 'side'
-    })
-  )
-  .add(
-    'no visible label',
-    () => render({
-      label: null,
-      'aria-label': 'Password',
-      description: 'Password must be at least 8 characters.'
-    })
-  )
-  .add(
-    'custom width',
-    () => render({
-      label: 'Password',
-      description: 'Password must be at least 8 characters.',
-      width: '100px'
-    })
-  )
-  .add(
-    'custom width, labelPosition: side',
-    () => render({
-      label: 'Password',
-      description: 'Password must be at least 8 characters.',
-      width: '440px',
-      labelPosition: 'side'
-    })
-  );
+type HelpTextStory = ComponentStoryObj<typeof TextField>;
+
+export default {
+  title: 'HelpText',
+  component: TextField,
+  args: {
+    label: 'Password',
+    description: 'Password must be at least 8 characters.'
+  }
+} as ComponentMeta<typeof TextField>;
+
+export let Default: HelpTextStory = {
+  name: 'description'
+};
+
+export const ErrorMessage = {
+  ...Default,
+  args: {
+    errorMessage: 'Create a password with at least 8 characters.',
+    validationState: 'invalid'
+  },
+  name: 'error message'
+};
+
+export const Disabled = {
+  ...Default,
+  args: {
+    isDisabled: true
+  },
+  name: 'disabled'
+};
+
+export const LabelAlignEnd = {
+  ...Default,
+  args: {
+    labelAlign: 'end'
+  },
+  name: 'labelAlign: end'
+};
+
+export const LabelPositionSide = {
+  ...Default,
+  args: {
+    labelPosition: 'side'
+  },
+  name: 'labelPosition: side'
+};
+
+export const NoVisibleLabel = {
+  ...Default,
+  args: {
+    label: null,
+    'aria-label': 'Password'
+  },
+  name: 'no visible label'
+};
+
+export const CustomWidth = {
+  ...Default,
+  args: {
+    width: '100px'
+  },
+  name: 'custom width'
+};
+
+export const CustomWidthLabelPositionSide = {
+  ...Default,
+  args: {
+    width: '440px',
+    labelPosition: 'side'
+  },
+  name: 'custom width, labelPosition: side'
+};
 
-function render(props: SpectrumTextFieldProps = {}) {
-  return (
-    <TextField label="Password" {...props} />
-  );
-}
+export const ContainerWithTextAlignmentSet = {
```

Comment:
This story was in storybook and since it wasn't interactive I moved it to Chromatic.